changelog.d/+update-contributing-md.md

```diff
@@ -0,0 +1 @@
+Update contributing.md
```

Comment:
we need this changelog entry for this PR?

Reply:
Whoops, fixed